src/services/worker/backupUtils.ts

```
@@ -383,66 +383,70 @@ const dbGetTableData = async () => {
+    for await (const talk of talks) {
+      const dbSchedule = await appDb.sched.get(talk.weekOf);
 
-    if (dbSchedule) {
-      const tmpSched = talk;
+      if (dbSchedule) {
+        const tmpSched = talk;
 
-      delete tmpSched.recipient;
-      delete tmpSched.sender;
-      delete tmpSched.weekOf;
+        delete tmpSched.recipient;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)